cedar-policy-generators/Cargo.toml

```diff
@@ -5,7 +5,8 @@ version = "4.0.0"
 publish = false
 
 [dependencies]
-arbitrary = "1.3"
+arbitrary = "=1.3.2"
+derive_arbitrary = "=1.3.2"
```

Comment:
Probably makes sense to add the same comment added in https://github.com/cedar-policy/cedar-spec/pull/464/commits/cbb8ac9749dbf932fac7eb93337f3038560ba573

Reply:
Done.